Dockerfile

```diff
@@ -1,6 +1,13 @@
 FROM python:3
 
-RUN pip install apprise j2cli
+RUN set -x \
+    \
+    && apt-get update \
+    && apt-get install -y --no-install-recommends jq \
+    && rm -rf /var/lib/apt/lists/* \
+    \
+    && pip install --no-cache-dir apprise j2cli
```

Comment:
I am used to remove pip cache from docker image to reduce space usage, but I can re-enable it in case it is needed.